helm/chart/maesh/Chart.yaml

```
@@ -25,5 +25,5 @@ dependencies:
     version: 0.0.3
     condition: tracing.deploy
   - name: metrics
-    version: 0.0.6
+    version: 0.1.0
```

See my previous comment.

```suggestion
    version: 0.0.6
```